linux/riscv64.config-patch

```diff
@@ -773,15 +780,23 @@
  CONFIG_SOC_TI=y
  
  #
-@@ -8383,6 +8587,7 @@
+@@ -8447,6 +8657,7 @@
  #
  # DEVFREQ Drivers
  #
 +# CONFIG_ARM_SUN8I_A33_MBUS_DEVFREQ is not set
  CONFIG_PM_DEVFREQ_EVENT=y
  CONFIG_EXTCON=y
  
-@@ -9057,8 +9262,10 @@
+@@ -8609,6 +8820,7 @@
+ CONFIG_PALMAS_GPADC=m
+ CONFIG_RICHTEK_RTQ6056=m
+ CONFIG_SD_ADC_MODULATOR=m
++# CONFIG_SUN20I_GPADC is not set
```

Comment:
Looks like this is a driver for D1, should we enable it?

Reply:
done